.github/dependabot.yml

```diff
@@ -1,5 +1,10 @@
 version: 2
 updates:
+- package-ecosystem: github-actions
+  directory: "/"
```

Comment:
Is the directory correct here? Asking because our workflows are located in: https://github.com/osmosis-labs/osmosis/tree/main/.github/workflows

Reply:
Yes, based on https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot, this is correct